README.md

```diff
@@ -527,9 +527,9 @@ A continuación realizar la transformación a POM, por medio de los siguientes p
        cy.visit(this.loginURL);
      }
 
-     public signIn(): void {
-       cy.get(this.userNameField).type("standard_user");
-       cy.get(this.passwordField).type("secret_sauce");
+     public signIn(In_User: string, In_Password: string): void {
```

Comment:
according to js coding standards variables and input parameters should be written in camelCase:

- userName or user
- pass, password, userPass, etc